artifactory/services/buildinfo.go

```diff
@@ -68,7 +64,7 @@ func (bis *BuildInfoService) GetBuildInfo(params BuildInfoParams) (pbi *buildinf
 	// Get build-info json from Artifactory.
 	httpClientsDetails := bis.GetArtifactoryDetails().CreateHttpClientDetails()
 
-	restApi := path.Join("api/build/", name, number)
+	restApi := path.Join("api/build/"+bis.getProjectQueryParam(params.ProjectKey), name, number)
```

Comment:
```suggestion
	restApi := path.Join("api/build/", name, number) + bis.getProjectQueryParam(params.ProjectKey)
```


If I'm not wrong, getProjectQueryParam returns the REST parameter.
However, it looks like in this case, the REST will be: 
`api/build/?buildRepo=<build-repo>/<build-name>/<build-number>`
But I think the REST should be:
`api/build/<build-name>/<build-number>?buildRepo=<build-repo>`